samples/scenarios/extract_only/sqlite_extract/dialect.xml

```diff
@@ -1,5 +1,1251 @@
+    <function group='operator' name='&gt;' return-type='bool'>
+      <formula>(%1 &gt; %2)</formula>
+      <argument type='str' />
+      <argument type='str' />
+    </function>
+    <function group='operator' name='&gt;' return-type='bool'>
+      <formula>(%1 &gt; %2)</formula>
+      <argument type='datetime' />
+      <argument type='datetime' />
+    </function>
+    <function group='operator' name='&gt;' return-type='bool'>
+      <formula>(%1 &gt; CAST(%2 AS TIMESTAMP))</formula>
+      <argument type='datetime' />
+      <argument type='date' />
+    </function>
+    <function group='operator' name='&gt;' return-type='bool'>
+      <formula>(CAST(%1 AS TIMESTAMP) &gt; %2)</formula>
+      <argument type='date' />
+      <argument type='datetime' />
+    </function>
+    <function group='operator' name='&gt;' return-type='bool'>
+      <formula>(%1 &gt; %2)</formula>
+      <argument type='date' />
+      <argument type='date' />
+    </function>
+    <function group='operator' name='&gt;=' return-type='bool'>
+      <formula>(%1 &gt;= %2)</formula>
+      <argument type='real' />
+      <argument type='real' />
+    </function>
+    <function group='operator' name='&gt;=' return-type='bool'>
+      <formula>(%1 &gt;= %2)</formula>
+      <argument type='str' />
+      <argument type='str' />
+    </function>
+    <function group='operator' name='&gt;=' return-type='bool'>
+      <formula>(%1 &gt;= %2)</formula>
+      <argument type='datetime' />
+      <argument type='datetime' />
+    </function>
+    <function group='operator' name='&gt;=' return-type='bool'>
+      <formula>(%1 &gt;= CAST(%2 AS TIMESTAMP))</formula>
+      <argument type='datetime' />
+      <argument type='date' />
+    </function>
+    <function group='operator' name='&gt;=' return-type='bool'>
+      <formula>(CAST(%1 AS TIMESTAMP) &gt;= %2)</formula>
+      <argument type='date' />
+      <argument type='datetime' />
+    </function>
+    <function group='operator' name='&gt;=' return-type='bool'>
+      <formula>(%1 &gt;= %2)</formula>
+      <argument type='date' />
+      <argument type='date' />
+    </function>
+    <function group='operator' name='^^' return-type='real'>
+      <formula>(CASE WHEN %1 &lt; 0 AND FLOOR(%2) &lt;&gt; %2 THEN NULL ELSE POW(%1,%2) END)</formula>
+      <argument type='real' />
+      <argument type='real' />
+    </function>
+    <function group='operator' name='^^' return-type='real'>
+      <formula>POW(%1,%2)</formula>
+      <argument type='real' />
+      <argument type='int' />
+    </function>
+    <function group='operator' name='^^' return-type='real'>
+      <formula>CAST(%1 AS FLOAT) ^ %2</formula>
+      <argument type='int' />
+      <argument type='int' />
+    </function>
+    <function group='operator' name='||' return-type='bool'>
+      <formula>(%1 OR %2)</formula>
+      <argument type='bool' />
+      <argument type='bool' />
+    </function>
+    <date-function name='DATEADD' return-type='datetime'>
```

Comment:
Missed this the first time.  For all of these date-function ones the export tool doesn't output the arguments correctly, they are all off by %1 is always the date-part-group value: https://tableau.github.io/connector-plugin-sdk/docs/dialect.  The 'sunday' in each of these is also erroneous.

Check out my recent conversion review for some working examples or the postgres example.

Its entirely possible some (or possibly all) are not even needed for extract-only, but I don't know that for sure, you might try a manual test.

Reply:
My manual test process: Create a calculation field for each date function, copy and paste some test cases from tdvt, for example `DATETRUNC('day', DATETIME([Datetime0]), 'monday')` for DATETRUNC, the calculation fields' result are all same with tdvt's standard result, also same with the result of using original dialect(CDataExtractDialect as base).

so I think it is ok to remove all date functions, thanks for pointing this out.